test/default-ilp-client-test.ts

```diff
@@ -22,10 +22,10 @@ describe('Default ILP Client', function(): void {
     const client = fakeSuceedingNetworkClient()
 
     // WHEN the balance for an account is requested
-    const amount = await client.getBalance('test.foo.bar')
+    const amount = await client.getBalance('test.foo.bar', 'Bearer blah')
```

Comment:
I think it might be worth keeping the non-bearer token case as well since it is an optional argument

Reply:
Woops, forgot to take it out of one test.